package.json

```diff
@@ -219,7 +219,7 @@
     "terser-webpack-plugin": "^5.1.4",
     "ts-lit-plugin": "^1.2.1",
     "ts-mocha": "^8.0.0",
-    "typescript": "^4.3.5",
+    "typescript": "4.3.5",
```

Comment:
why pin to exact version ?

Reply:
Will bump it in future PR, but higher version is not compatible yet